lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_generate_po_file_from_metadata.rb

```diff
@@ -0,0 +1,136 @@
+require_relative '../../helper/po_extended'
+require 'fastlane/action'
+require 'gettext/po'
+
+module Fastlane
+  module Actions
+    class AndroidGeneratePoFileFromMetadataAction < Action
+
+      # SPECIAL_KEYS are keys that need to be treated specially
+      SPECIAL_KEYS = %w[release_notes release_notes_previous].freeze
+      REQUIRED_KEYS_TO_COMMENT_HASH = {
+        full_description: 'Multi-paragraph text used to display in the Play Store. Limit to 4000 characters including spaces and commas!',
+        title: 'Title to be displayed in the Play Store. Limit to 30 characters including spaces and commas!',
+        short_description: 'Short description of the app to be displayed in the Play Store. Limit to 80 characters including spaces and commas!',
+        release_notes_short: nil,
+        release_notes: 'Release notes for this version to be displayed in the Play Store. Limit to 500 characters including spaces and commas!',
+        release_notes_previous: nil
```

Comment:
> I'm not sure to see what the user could do in such a case. Like, if the release_notes_previous are missing, since it's not a big deal, what would be the benefit in aborting? Trying to fix the issue manually before retrying? But then what would the user do, create an empty release_notes_previous.txt file just to circumvent the issue? Trying to dig the previous release notes from the previous git tag to copy/paste them? Is that worth the effort in delaying the release process for something that is not really mandatory to be able to continue the release of current version N?

I was thinking that yes, the user might want to dig them up from git somehow. A big effort for a little benefit. Like you say [considering that they are not mandatory](https://github.com/wordpress-mobile/release-toolkit/pull/414#discussion_r993642010) it's not worth it.

Reply:
> `release_notes_previous` should not be required. If the `release_notes_previous.txt` file is present, then the code should use it as the entry for it, but if it's not present, that shouldn't be a hard error.
> 
> We should just maybe warn for it in the logs (`UI.warning`) but then continue and generate the `.po` file (just without an entry for the previous release notes if the file is missing)

✅  by 00ee2c7e78034ab86b1e4621afddb9614dfb8c0b